### PR TITLE
Last change to replace typehead usage with at.js broke the dropdown users list

### PR DESCRIPTION
**Pull request by xillibit**
Pull Request for Issue # .

### CODE REVIEW

**Comment by 810** [comment]
when i click on the scrollbar to scroll to bottom, then the menu is gone. need to be active

File: src/components/com_kunena/template/crypsisb3/assets/js/search.js [hunk 1]
```diff
@@ -15,17 +15,17 @@ jQuery(document).ready(function($) {
 
 		$('#kusersearch').atwho({
 			at: "",
-			tpl: '<li data-value="${username}"><span class="glyphicon glyphicon-user"></span> ${username} <small>(${name})</small></li>',
+			displayTpl: '<li data-value="${name}"><span class="icon icon-user"></span> ${name} <small>(${name})</small></li>',
```
Comment (810) [inline src/components/com_kunena/template/crypsisb3/assets/js/search.js]: need to be glyphicon

Comment (xillibit) [reply src/components/com_kunena/template/crypsisb3/assets/js/search.js]: which glyphicon should need to use ?

Comment (810) [reply src/components/com_kunena/template/crypsisb3/assets/js/search.js]: glyphicon glyphicon-user

**Pull request merged**